src/editor/navigation.ts

```diff
@@ -1,7 +1,16 @@
-import { ExtensionContext, Position, Range, Uri, commands, window } from 'vscode';
+import { ExtensionContext, Position, Range, TextEditor, Uri, commands, window } from 'vscode';
 import { ExtensionApi } from '../backend/api';
 import { DiagnosticReport } from '../backend/types';
 
+const reportStepDecorationType = window.createTextEditorDecorationType({
+    light: {
+        backgroundColor: '#eeb'
```

Comment:
It would be nice if these were moved into a contributed [ThemeColor](https://code.visualstudio.com/api/references/theme-color), so it's customizable by users.